recipes/scikit-learn/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "0.19.0" %}
```

Comment:
Is it alright if we stick to the content in `master` of the `scikit-learn` feedstock for now? I'm pretty sure `staged-recipes` won't actually change the recipe content even if the recipe is different here (that is my experience at least), but I think it would just avoid some other potential issues if we make sure it matches `master` in the feedstock.